creation/lib/cgWParamDict.py

```diff
@@ -397,7 +408,7 @@ def populate(self, other=None):
             # TODO: This check could be done in the XML, checking if the entries are consistent in the current version
             # fetch the on-demand cvmfs provisioning feature setting
             # if on-demand CVMFS not used at the global level; ignore and continue
-            ondemand_cvmfs = self.dicts["attrs"].get("GLIDEIN_USE_CVMFSEXEC", 0)
+            ondemand_cvmfs = self.dicts["attrs"].get("GLIDEIN_USE_CVMFS", 0)
```

Comment:
Should we also check for `GLIDEIN_USE_CVMFSEXEC` for backward compatibility?

Reply:
The configuration knob `GLIDEIN_USE_CVMFSEXEC` will be changed to `GLIDEIN_USE_CVMFS`, but I do not know for sure if we need to ensure backward compatibility in this case, if this change will eventually be propagated to production. 

@mambelli would like to know your thoughts on this!